components/codec/src/byte.rs

```diff
@@ -26,6 +29,29 @@ impl MemComparableByteCodec {
         (src_len / MEMCMP_GROUP_SIZE + 1) * (MEMCMP_GROUP_SIZE + 1)
     }
 
+    /// Gets the first encoded bytes' length in memory-comparable-encoded data.
+    ///
+    /// Memory-comparable-encoding includes a VarInt encoded length prefix (1 ~ 9 bytes) and N bytes payload.
+    /// This function gets the total bytes length of memory-comparable-encoded data, including the length prefix.
+    ///
+    /// Note:
+    ///     - This function won't check whether the bytes are encoded correctly.
+    ///     - There can be multiple memory-comparable-encoded data, placed one by one. This function only returns
+    ///       the length of the first one.
+    pub fn encoded_bytes_len(encoded: &[u8], desc: bool) -> usize {
```

Comment:
Looks like this function is not used?

Reply:
this function is a replacement for `crate::util::codec::bytes::encoded_bytes_len`, which used in 
https://github.com/tikv/tikv/blob/cb0e3046ca315ce00964b9bf11bd210c08f4ccdf/src/coprocessor/codec/datum.rs#L1013

Reply:
Got it.